packages/g/gum/package.yml

```diff
@@ -0,0 +1,33 @@
+name       : gum
+version    : 0.13.0
+release    : 1
+source     :
+    - https://github.com/charmbracelet/gum/archive/refs/tags/v0.13.0.tar.gz : 329a38f3453b4be1f00e1fcb987aacf574fe3a8cc592084529c05716ddf4e7c4
+homepage   : https://charm.sh
+license    : MIT
+component  : programming.tools
+summary    : A tool for glamorous shell scripts
+description: |
+  Effects include
+  choose     Choose an option from a list of choices
+  confirm    Ask a user to confirm an action
+  file       Pick a file from a folder
+  filter     Filter items from a list
+  format     Format a string using a template
+  input      Prompt for some input
+  join       Join text vertically or horizontally
+  pager      Scroll through a file
+  spin       Display spinner while running a command
+  style      Apply coloring, borders, spacing to text
+  table      Render a table of data
+  write      Prompt for long-form text
+  log        Log messages to output
+networking : yes
+builddeps  :
+    - golang
+build      : |
+    go build -v
```

Comment:
I built this locally and noticed a few issues, or missing features compared to the Arch package.
- `gum -v` listed an unknown version
- gum has a man page but it's not built/provided
- the GOFLAGS weren't set (the ones in go-task/other Go packages are used for a reason, they strip build paths from panic traces amongst other things).
- No shell completions provided

So, since I really wanted to try this out myself I just went ahead and solved all of these by using bits from our other recipes and the Arch one.
```
build      : |
    source_date=$(date -d "@$SOURCE_DATE_EPOCH" +"%Y-%m-%d")
    export GOFLAGS="-buildmode=pie -trimpath -mod=readonly -modcacherw "

    # mkdir build
    go build -v -o build/gum \
        -ldflags="-linkmode=external -X=main.Version=$version"

    # Build man page
    ./build/gum man > build/gum.1
    # Reproducible man page date
    sed \
        -i build/gum.1 \
        -e "s|\"[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}\"|\"$source_date\"|"

    # Completions
    for shell in bash fish zsh; do
        ./build/gum completion "$shell" > "build/$shell-completion"
    done
install    : |
    install -Dm00755 build/gum $installdir/usr/bin/gum
    install -Dm00644 build/gum.1 -t $installdir/usr/share/man/man1/

    # Completions
    install -Dm00644 build/bash-completion $installdir/usr/share/bash-completion/completions/gum
    install -Dm00644 build/zsh-completion $installdir/usr/share/zsh/site-functions/_gum
    install -Dm00644 build/fish-completion $installdir/usr/share/fish/vendor_completions.d/gum.fish

    install -Dm00644 README.md $installdir/usr/share/docs/gum/README.md
```
Feel free to copy it in verbatim, or make further modifications. Up to you.